server/routes/activities/record-attendance/handlers/attendanceList.ts

```diff
@@ -93,6 +98,75 @@ export default class AttendanceListRoutes {
     })
   }
 
+  GET_ATTENDANCES = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { selectedInstanceIds } = req.session.recordAttendanceRequests
+
+    const selectedSessions = new Set<TimeSlot>()
```

Comment:
Allow the view to see which sessions have been selected, i.e. AM, PM and / or ED